feature-pack/src/license/full-feature-pack-licenses.xml

```diff
@@ -603,7 +603,7 @@
       </licenses>
     </dependency>
     <dependency>
-      <groupId>io.smallrye</groupId>
+      <groupId>io.smallrye.config</groupId>
```

Comment:
Please re-alphabetize. (I think it's important as EAP productization files PRs to fix it any time it's not in order.)

Reply:
done